services/workflows-service/src/ui-definition/ui-definition.repository.ts

```diff
@@ -40,11 +40,16 @@ export class UiDefinitionRepository {
     uiContext: keyof typeof UiDefinitionContext,
     args: Prisma.SelectSubset<T, Omit<Prisma.UiDefinitionFindFirstOrThrowArgs, 'where'>>,
```

Comment:
you can use args for fetch by id, just edit to have to pick the id property from the where prima arg object

Reply:
mit<Prisma.UiDefinitionFindFirstOrThrowArgs, 'where'> means, without where...
I can change it to include where but then I'll have to add where everythwere in the code